vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -472,8 +482,7 @@ switch ($env:ProvisioningPhase) {
         Update-Registry -containerRuntime $containerRuntime
         Get-ContainerImages -containerRuntime $containerRuntime -windowsSKU $windowsSKU
         Get-FilesToCacheOnVHD -containerRuntime $containerRuntime
-        # Show disk space
-        Get-CimInstance -ClassName Win32_LogicalDisk
```

Comment:
In which case this does not work?

Reply:
No idea. I just found that it does not log the disk info as expected sometimes.

Reply:
@mainred do you have other concern for this PR and need me to update it?